app/src/main/java/com/kickstarter/ui/activities/DeepLinkActivity.kt

```diff
@@ -87,6 +94,19 @@ class DeepLinkActivity : BaseActivity<DeepLinkViewModel.ViewModel?>() {
         finish()
     }
 
+    private fun startProjectActivityForSave(uri: Uri, isFfEnabled: Boolean) {
```

Comment:
picky comment, can we move this method as an extension method to `ActivityExt.kt`